src/main/windows/windowManager.ts

```diff
@@ -258,14 +258,23 @@ export class WindowManager {
     isResizing = false;
 
     handleWillResizeMainWindow = (event: Event, newBounds: Electron.Rectangle) => {
-        log.silly('WindowManager.handleWillResizeMainWindow');
+        log.debug('WindowManager.handleWillResizeMainWindow');
```

Comment:
Can we switch this back to `silly`? It's going to generate a lot of noise for people who want just debug logging of events.